crates/joystick/src/windows.rs

```diff
@@ -1,13 +1,5 @@
-// THIS WILL NOT WORK UNTIL ITS IMPLEMENTED FOR WINDOWS
-
-use evdev::{
-    uinput::{VirtualDevice, VirtualDeviceBuilder},
-    AbsInfo, AbsoluteAxisCode, AttributeSet, EventType, InputEvent, KeyCode, UinputAbsSetup,
-};
-
-use std::sync::Arc;
-use std::sync::Mutex;
 use std::time::{Duration, Instant};
+use vigem_client::{Client, XButtons, Xbox360Wired};
 
 static TAP_DURATION: u64 = 50;
```

Comment:
Note that this (and `KeyAction`) is shared between windows/linux. Might be good to have them in one place (a `common.rs`?)

Reply:
absolutely, will add a note to my trait task in #7